playbooks/roles/wordpress/tasks/nginx.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: open HTTPS port on the firewall
+  ufw:
+    rule: allow
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
+    - 443
+
+- name: mount /var/www
+  mount: name=/var/www src=/dev/vdb1 fstype=ext4 state=mounted
+
+- name: chown www-data /var/www
+  file: path=/var/www mode=750 owner=www-data group=www-data state=directory
+
+- name: mkdir /var/www/log
+  file: path=/var/www/log mode=750 state=directory
+
+- name: Ensure directory for config files exists
+  file:
+    path: /etc/nginx/sites-available/wordpress
+    state: directory
+
+- name: Copy nginx global configuration
+  synchronize: src="global/" dest=/etc/nginx/global checksum=yes delete=yes owner=no group=no
+
+- name: Add landing site nginx configuration
+  template:
+    src: ../templates/wordpress-nginx.j2
+    dest: /etc/nginx/sites-available/wordpress/{{ LANDING_SITE_HOSTNAME }}.conf
+
+- name: Add default site nginx configuration
+  template:
+    src: ../templates/default-nginx.j2
+    dest: /etc/nginx/sites-available/wordpress/{{ DEFAULT_SITE_HOSTNAME }}.conf
```

Comment:
@Agrendalath, nit: does this default site have to be under the `wordpress` sub-directory since it has nothing to do with Wordpress. Ditto for the maintenance site.

Reply:
Done.